scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -778,3 +778,11 @@ def test_quat_ownership():
 
     r._quat[0] = np.array([0, -1, 0, 0])
     assert_allclose(s._quat[0], np.array([1, 0, 0, 0]))
+
+
+def test_estimate_no_rotation():
```

Comment:
Add a test with non-trivial rotation, two vectors and many vectors, without noise and with noise. Obviously with noise you won't get the rotation back precisely, but you can control the level of error, especially if you compute the covariance matrix.